docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -390,7 +390,8 @@ spring:
   cloud:
     config:
       server:
-        foo: bar
+        overrides:
```

Comment:
The documentation says: ```To declare overrides just add a map of name-value pairs to spring.cloud.config.server.overrides```. I thought the correct would be to have the overrides map inside the `overrides` key. Isn't that true?